server/s3_put_object_action.cc

```diff
@@ -58,7 +58,20 @@ S3PutObjectAction::S3PutObjectAction(
   old_object_oid = {0ULL, 0ULL};
   old_layout_id = -1;
   new_object_oid = {0ULL, 0ULL};
-
+  fault_mode_active = false;
+  last_object_size = 0;
+  primary_object_size = 0;
+  total_object_size_consumed = 0;
+  no_of_blocks_written = 0;
+  create_fragment_when_write_success = false;
+
+  // Default to 10 objects in S3 fault mode
+  max_objects_in_s3_fault_mode = 10;
```

Comment:
I think we should make this 10 value as const in the header file

Reply:
ok